packages/ui/src/common/components/Tooltip/Tooltip.tsx

```diff
@@ -94,6 +95,7 @@ export const Tooltip = ({
                 {...attributes.popper}
                 {...popUpHandlers}
                 isTooltipActive={isTooltipActive}
+                forBig={forBig}
```

Comment:
Maybe we could use the [arrow](https://popper.js.org/docs/v2/modifiers/arrow/) from popper?